tqec/app.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Google LLC
```

Comment:
`tqec` is not a Google project.
```suggestion
# Copyright 2023 tqec contributors
```

Reply:
That was copied in by mistake from a tutorial.

Can we remove all the legal boilerplate from the files, since the `LICENSE` is present for the entire repo?

Reply:
Yeah, let's do that.